README.md

```diff
@@ -9,26 +9,42 @@ A practical functional library for Javascript programmers.
 [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/ramda/ramda?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 
-Goals
------
+Why Ramda?
+----------
 
 <img src="http://ramda.jcphillipps.com/logo/ramdaFilled_200x235.png" 
-     width="200" height="235" align="left" hspace="12" />
-Using this library should feel as much like using Javascript as 
-possible.  Of course it's functional Javascript, but we're not 
-introducing lambda expressions in strings, we're not borrowing consed 
+     width="200" height="235" align="right" hspace="12" />
+
+Ramda makes it simple for you to build complex logic through
+functional composition. This means sticking simple functions together.
+It is designed to help you get you job more elegantly with simpler code.
+Ramda does not mutate your state. Any data you throw at Ramda is safe from
+hidden state changes, and this is a very powerful weapon to battle complexity.
+
+[Compare Ramda to other FP libraries](http://www.google.com)
+
+While LoDash or Underscore are great libraries, they sometimes introduce
+complexities in your code which Ramda avoids with a simpler design.
+Also see [Why Ramda?](http://fr.umio.us/why-ramda/) and [Hey Underscore](https://www.youtube.com/watch?v=m3svKOdZijA&app=desktop).
+
+
+Philosophy
+----------
+Using this library feels much like using Javascript, for good.
+Of course it's functional Javascript, but we're not introducing
+lambda expressions in strings, we're not borrowing consed 
 lists, we're not porting over all of the Clojure functions.
 
-Our basic data structures will be normal Javascript objects, and our 
-usual collections will be Javascript arrays.  We will not try to reach 
+Our basic data structures are normal Javascript objects, and our 
+usual collections are Javascript arrays.  We do not try to reach 
 the point where all the functions have only zero, one, or two arguments.
-We will certainly try to keep some of the normal features of Javascript 
+We certainly keep some of the normal features of Javascript 
 that seem to be unusual in functional languages, including variable 
 length function signatures and functions as objects with properties.
```

Comment:
The line about variadic functions has turned out not really to be true.

The copy you're editing is so old that it represents initial dreams more than settled fact.  Feel free to strip out things like that that don't reflect current reality.